packages/core/src/styles/clr-tokens.map.scss

```diff
@@ -0,0 +1,878 @@
+:root,
+[cds-theme] {
```

Comment:
Technically this is `@clr/ui` code dependent on `@cds/core` code in order of dependencies so I'm not 100% sure if it should be in core for this reason. However I could see it categorized as a custom theme of Core. Maybe we could name it like the other themes, something like `theme.clr-ui.scss`?

Also could we add a comment block at the top of this file describing that this is a shim/theme only for the use of apps that use `@clr/ui` currently and new apps should not import this?

Reply:
@Shijir where are we here?